src/dfs_sdk/scaffold.py

```diff
@@ -21,8 +21,8 @@
 TNT = re.compile(r'datera_tenant_id\s+?=\s+?(?P<tenant_id>.*)')
 LDP = re.compile(r'datera_ldap_server\s+?=\s+?(?P<ldap>.*)')
 
-LATEST = "2.2"
-FALLBACK = ["2", "2.1"]
+LATEST = "2.3"
+FALLBACK = ["2", "2.1", "v2.2"]
```

Comment:
Can you check if the FALLBACK should include "v2.2" or "2.2" ?

Reply:
FALLBACK should be '2.2'. I've pushed the change. Please review!